build-tools/create-dotnet-msi/Microsoft.NET.Workload.Android.wix.in

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
+
+  <Product Name="Microsoft.NET.Workload.Android"
+      Id="*"
+      Language="1033"
+      Version="@MSIVERSION@"
+      Manufacturer="Microsoft"
+      UpgradeCode="86a83cc9-6b75-489f-be10-27988df907c6">
```

Comment:
Where'd this value come from, if anywhere?  What's it used for?

We might need links to relevant WiX docs.

Reply:
This is a Guid that is a unique identifier for the MSI, best docs I can find:

https://docs.microsoft.com/windows/win32/msi/upgradecode

I created this file originally from a File -> New WIX Setup Project in Visual Studio, and it generates a Guid for you.

Reply:
Should we add an XML comment to the `.wix.in` file mentioning this, or leave it in the commit message?

```xml
<!--
    DO NOT CHANGE UpgradeCode; see https://docs.microsoft.com/windows/win32/msi/upgradecode
  -->
<Product Name=…
```

Reply:
I think the commit message is fine, I doubt we will change `.wix.in` much once it is working?